sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/RequestData.cs

```diff
@@ -65,5 +64,32 @@ internal static bool IsSuccess(Activity activity, string? responseCode, Operatio
                 return activity.Status != ActivityStatusCode.Error;
             }
         }
+
+        private void SetHttpRequestPropertiesAndResponseCode(Activity activity, ref AzMonList httpTagObjects, out string responseCode)
+        {
+            Url = httpTagObjects.GetRequestUrl().Truncate(SchemaConstants.RequestData_Url_MaxLength);
+            Name = TraceHelper.GetOperationName(activity, ref httpTagObjects).Truncate(SchemaConstants.RequestData_Name_MaxLength);
+            responseCode = AzMonList.GetTagValue(ref httpTagObjects, SemanticConventions.AttributeHttpStatusCode)
+                                                ?.ToString().Truncate(SchemaConstants.RequestData_ResponseCode_MaxLength)
+                                                ?? "0";
+        }
+
+        private void SetHttpV2RequestPropertiesAndResponseCode(Activity activity, ref AzMonList httpTagObjects, out string responseCode)
+        {
+            Url = httpTagObjects.GetNewSchemaRequestUrl().Truncate(SchemaConstants.RequestData_Url_MaxLength);
+            Name = TraceHelper.GetNewSchemaOperationName(activity, Url, ref httpTagObjects).Truncate(SchemaConstants.RequestData_Name_MaxLength);
+            responseCode = AzMonList.GetTagValue(ref httpTagObjects, SemanticConventions.AttributeHttpResponseStatusCode)
+                                    ?.ToString().Truncate(SchemaConstants.RequestData_ResponseCode_MaxLength)
+                                    ?? "0";
+        }
+
+        private void SetMessagingRequestPropertiesAndResponseCode(Activity activity, ref AzMonList messagingTagObjects, out string responseCode)
+        {
+            Url = AzMonList.GetTagValue(ref messagingTagObjects, SemanticConventions.AttributeMessagingUrl)?.ToString().Truncate(SchemaConstants.RequestData_Url_MaxLength);
+            Name = TraceHelper.GetOperationName(activity, ref messagingTagObjects).Truncate(SchemaConstants.RequestData_Name_MaxLength);
```

Comment:
Set this to `activity.DisplayName`. GetOperationName is not applicable to messaging

Reply:
Same here, will get it removed. Maybe slight change from original implementation.